### PR TITLE
Create a Shell for easy to add user

### PR DESCRIPTION
**Pull request by wade0317**

### CODE REVIEW

**Comment by wade0317** [comment]
Readme for Manager user will update after pull request is closed.

File: vpnadduser.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{
+
+	while  [[ "$VPN_USER" == "" ]]
+	do
+		echo  -e "VPN USER:\c" ; VPN_USER= ; read VPN_USER
+	done
+
+	while  [[ "$VPN_PASSWORD" == "" ]]
+	do
+		echo  -e "VPN PASSWORD:\c" ; VPN_PASSWORD= ; read VPN_PASSWORD
```
Comment (ad-m) [inline vpnadduser.sh]: I would just like to generate a password.

File: vpnsetup.sh [hunk 2]
```diff
@@ -24,9 +24,9 @@
 # - All values MUST be placed inside 'single quotes'
 # - DO NOT use these special characters within values: \ " '
 
-YOUR_IPSEC_PSK=''
-YOUR_USERNAME=''
-YOUR_PASSWORD=''
+YOUR_IPSEC_PSK='greatvpn'
```
Comment (ad-m) [inline vpnsetup.sh]: Why??

**Comment by hwdsl2** [comment]
@wade0317 Thank you for contributing! We'll plan to instead look into alternative approaches for adding multiple VPN users that are better integrated with the setup scripts.

**Pull request closed without merging**